modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -2,17 +2,17 @@
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 [id='virt-importing-vm-prerequisites_{context}']
-= Prerequisites for importing a virtual machine into {VirtProductName}
+= Prerequisites for importing a virtual machine
 
 ifdef::virt-importing-rhv-vm[]
-Importing a Red Hat Virtualization virtual machine into {VirtProductName} has the following prerequisites:
+Importing a virtual machine from Red Hat Virtualization (RHV) into {VirtProductName} has the following prerequisites:
 
-* If the operating system of the virtual machine does not map to a common {VirtProductName} template, you must create a custom ConfigMap to map the operating system to a common {VirtProductName} template.
-
-* If you are importing a VM with a Windows operating system, you must create a Windows {VirtProductName} template and a custom ConfigMap.
-
-* The default storage class must be NFS.
+ifeval::["{VirtVersion}" == "2.4"]
+* If you are importing a Linux VM that does not have a corresponding {product-title} template, you must create a custom ConfigMap.
```

Comment:
Please remove Linux, as this is true for any VM OS.
Also if there is no such common template in CNV, need to create a custom template, and to create a configMap to map the VM OS to it.

Reply:
But we do not provide instructions for creating a custom template for Linux OS, only Windows.

I understand that **if** there is no CNV template for Linux OS, they can map to fedora31. Is that not the case?